web_src/js/components/DashboardRepoList.vue

```diff
@@ -255,6 +256,40 @@ const sfc = {
     $(el).find('.dropdown').dropdown();
     nextTick(() => {
       this.$refs.search.focus();
+      this.$refs.search.addEventListener('keydown', (e) => {
+        const previousPageLink = this.$refs.previousPageLink;
+        const nextPageLink = this.$refs.nextPageLink;
+        switch (e.key) {
+          case 'Enter':
+            this.$refs[`activeIndex${this.activeIndex}`][0].click();
```

Comment:
Why `[0]` here?

Could the `activeIndex???` be replace by CSS selector? just query the active item?

Reply:
This is to select repository, press Enter to enter repository

Reply:
Change to this?
```
document.querySelector('.repo-list-link.active').click();
```

Reply:
If it works (with a more specific selector), I think `querySelector` seems clearer, then no `ref` is needed.

Reply:
Modified!